### DISCUSSION TITLE
Google Colab PyBASA fails with following error

### DISCUSSION THREAD

**Discussion by ArnabKar** [yangheng95/PyABSA#373] [General]
Error:

```
[2023-12-18 22:37:43] (2.3.4) PyABSA(2.3.4): If your code crashes on Colab, please use the GPU runtime. Then run "pip install pyabsa[dev] -U" and restart the kernel.
Or if it does not work, you can use v1.x versions, e.g., pip install pyabsa<2.0 -U




WARNING: When you fails to load a checkpoint, e.g., Unexpected key(s),
Try to downgrade transformers<=4.29.0.
```

I tried the following:
- Use PyABSA 1.x
- Downgrade to Python 3.6
- "pip install pyabsa[dev] -U"
- Downgraded transformers, as suggested above

Same error persists regardless, and I used CPU, CPU HighMem, GPU machines. Still same error! 

Some help with this would be appreciated.

**Comment by sophiaaslam1** [comment]
Hi! I am also getting the same error and not sure why! Very urgent!

**Comment by surabhiwaingankar** [comment]
I am also facing the same error in the latest version - 2.4.1.post1, not sure why. Even I tried everything to fix it but it didn't work. So I used version 1.16.27 and that is working fine but I want to use the latest version as there are more features and it has detailed documentation available.

**Comment by Sohamiscurious** [comment]
Even facing the same issue, would love to have fix on this !
